libraries/standard/mqtt/cbmc/proofs/MQTT_GetIncomingPacketTypeAndLength/Makefile

```diff
@@ -1,9 +1,34 @@
+#
+# Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+
 HARNESS_ENTRY=harness
 HARNESS_FILE=MQTT_GetIncomingPacketTypeAndLength_harness
 
 DEFINES +=
 INCLUDES +=
 REMOVE_FUNCTION_BODY +=
+
+# The getRemainingLength loop is unwound 5 times because getRemainingLength()
+# divides a size_t variable by 128 until it zero to stop the loop.
```

Comment:
grammar: _...until it __reaches__ zero..._

Reply:
thanks